notebooks/samples/tensorflow/transfer_learning/ai_platform_transfer_learning.ipynb

```diff
@@ -0,0 +1,847 @@
+{
```

Comment:
<p>We need TF 2.1.0, remove tf-nightly</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/122/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
Addressed this!